src/sdk/pynni/nni/compression/torch/speedup/compressor.py

```diff
@@ -147,6 +147,7 @@ def infer_modules_masks(self):
                 # modules which are not data dependent (e.g., do
                 # not have conditionals on data in tensors)
                 # so, if a node is not traced, we just skip it.
+                _logger.info('Warning: %s not found in the traced graph, just skip it.', module_name)
```

Comment:
_logger.warning('%s has mask, but not found in the traced graph, just skip it.', module_name)